### ISSUE TITLE
Provide a way to return null (or other value) when an authz check failes

### ISSUE THREAD

**Issue by blevine** [AstrumU/graphql-authz#68]
I'd like to be able to return null (or other specified value) for a field when the authz check fails.

See https://github.com/AstrumU/graphql-authz/discussions/65#discussion-3866317

**Comment by velias** [comment]
Agree, there should be possibility to return `null` for the field with error noted in the error section of response, but return other fields still (aka partial response)
